helm/memcached-operator/config/manager/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+resources:
+- manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: quay.io/example-inc/memcached-operator
+  newTag: v0.0.1
```

Comment:
This being here means you ran `make deploy` before pushing. I think thats correct, but should we add that to the script?

Reply:
Actually, I think we ought to leave this out. The user will be creating their own images, so this should not point to a non-existent image.

Reply:
When they run the command to built with their image/tag it will be subscribed automatically.
however, I think it good we keep it in the script because:
- In a way, we are ensuring that the step suggested will work
- we ensure that the example has all manifests filled which is helpful to consult how it is without need to run
- if it sofers braking change we will get in the update 
- also, the image is used to gen the bundle